includes/WPGraphQLContentBlocks.php

```diff
@@ -117,9 +117,9 @@ function () {
 
 						echo sprintf(
 							'<div class="notice notice-error">' .
-							'<p>%s</p>' .
+								'<p>%s</p>' .
 							'</div>',
-							__( 'WPGraphQL Content Blocks appears to have been installed without it\'s dependencies. If you meant to download the source code, you can run `composer install` to install dependencies. If you are looking for the production version of the plugin, you can download it from the <a target="_blank" href="https://github.com/wpengine/wp-graphql-content-blocks/releases">GitHub Releases tab.</a>', 'wp-graphql' )
+							esc_html__( 'WPGraphQL Content Blocks appears to have been installed without it\'s dependencies. If you meant to download the source code, you can run `composer install` to install dependencies. If you are looking for the production version of the plugin, you can download it from the <a target="_blank" href="https://github.com/wpengine/wp-graphql-content-blocks/releases">GitHub Releases tab.</a>', 'wp-graphql-content-blocks' )
```

Comment:
I'm just eyeballing this, but won't `esc_html__` cause the link to not be clickable?

Reply:
re: broken link..

![image](https://github.com/wpengine/wp-graphql-content-blocks/assets/1254870/b0706a27-ac8f-4122-91eb-dce35f66bc2c)

Reply:
good catch! that should probably changed to a `wp_kses_*`. 
I had to switch back to client work for a bit, but Ive got `allow edits from maintainers` on if you don't want to way